lms/djangoapps/django_comment_client/utils.py

```diff
@@ -218,12 +224,12 @@ def get_discussion_category_map(course):
     return _filter_unstarted_categories(category_map)
 
 
-def get_discussion_categories_ids(course):
+def get_discussion_categories_ids(course, user):
```

Comment:
gah!  this seems to be yet another version of what we're using for listing user-accessible discussion ids, though this one again is generating the category_map and then fishing the keys out.  ok so maybe this can be changed to get the ids directly from modules and then the two places in the views can use this function?